docs/source/upgrade.rst

```diff
@@ -94,27 +94,31 @@ The same :class:`cupy.cuda.Stream` instance can now safely be shared between mul
 
 To achieve this, CuPy v10 will not destroy the stream (``cudaStreamDestroy``) if the stream is the current stream of any thread.
 
+Baseline API Update
+-------------------
+
+Baseline API has been bumped from NumPy 1.20 and SciPy 1.6 to NumPy 1.21 and SciPy 1.7.
+CuPy v10 will follow the upstream products’ specifications of these baseline versions.
```

Comment:
It seems Sphinx converts the ascii `'` to `’` (U+2019 right single quotation mark).

Reply:
```suggestion
CuPy v10 will follow the upstream products' specifications of these baseline versions.
```